javascript/javascript3/week2/resolveAfter.js

```diff
@@ -0,0 +1,38 @@
+/* Create a function that has one parameter: resolveAfter. Calling this function will
+ return a promise that resolves after the resolveAfter seconds has passed.
+When you have written the promise, use it with async/await */
+
+myFunction(8).then((message) => {
+    console.log(message); // logged out after 8 seconds
+})
+    .catch((error) => {
+        console.log(error);
+
+    });
+
+function myFunction(seconds) {
```

Comment:
Give the function a meaningful name. For example `delay`

Reply:
done